valkyrie/Gemfile

```diff
@@ -3,4 +3,4 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in valkyrie.gemspec
 gemspec
-gem 'rubocop', github: 'nelsonjr/rubocop', branch: 'feature/ignore-parent-exclude'
+gem 'rubocop', github: 'nelsonjr/rubocop', ref: 'a5db18a49b9f318644ae7209c73452fb7bbcdc49'
```

Comment:
Why the nelsonjr fork and not bbatsov?

Reply:
https://github.com/bbatsov/rubocop/pull/4329 is the feature we need (the --ignore-parent-excludes command hasn't been merged yet.)

Reply:
Can you put in a comment about that so we know when we can move off the fork?

Reply:
:+1: Done